crates/bevy_asset/src/asset_server.rs

```diff
@@ -171,13 +174,18 @@ impl AssetServer {
     /// Assets loaded with matching extensions will be blocked until the
     /// real loader is added.
     pub fn preregister_loader(&self, extensions: &[&str]) {
-        let mut loaders = self.server.loaders.write();
+        let mut loaders = self
+            .server
+            .loaders
+            .write()
+            .unwrap_or_else(std::sync::PoisonError::into_inner);
         let loader_index = loaders.len();
         for extension in extensions {
             if self
                 .server
                 .extension_to_loader_index
                 .write()
+                .unwrap_or_else(std::sync::PoisonError::into_inner)
```

Comment:
This is being used enough, please add it to the use statements to reduce the number of fully qualified names used.

Reply:
Can do!